apps/api/src/routes/community/community.ts

```diff
@@ -0,0 +1,141 @@
+import { Hono } from '@api/utils/hono';
+import { zValidator } from '@hono/zod-validator';
+
+import { handleError } from '@api/utils/errors';
+import { authMiddleware } from '@api/middlewares/auth';
+import {
+  fetchCommunityQuestion,
+  fetchCommunityQuestions,
+  handleAddQuestion,
+  handleDeleteCommentById,
+  handleDeleteCommentByQuestionId,
+  handleDeleteQuestion,
+  handleSubmitComment,
+  handleUpdateQuestion,
+  handleUpvote
+} from '@api/services/community';
+import {
+  ZCommunityComment,
+  ZCommunityQuestion,
+  ZCommunityQuestionDelete,
+  ZCommunityQuestions,
+  ZCommunityQuestionUpdate,
+  ZNewCommunityQuestion,
+  ZUpvoteComment
+} from '@cio/utils/validation/community';
+
+export const communityRouter = new Hono()
+  // --- get all community questions
+  .get('/questions', authMiddleware, zValidator('query', ZCommunityQuestions), async (c) => {
+    try {
+      const { orgId } = c.req.valid('query');
+
+      const result = await fetchCommunityQuestions(orgId);
+
+      return c.json({ success: true, data: result }, 201);
+    } catch (error) {
+      return handleError(c, error, 'Failed to load community questions');
+    }
+  })
+  // --- get single community question
+  .get('/question', authMiddleware, zValidator('query', ZCommunityQuestion), async (c) => {
```

Comment:
Instead of a query param, use a parameter `/question/{slug}`

Reply:
fixed